packages/marko-web-inquiry/translations.js

```diff
@@ -0,0 +1,10 @@
+const translations = {
+  en: {
+    firstNameLabel: 'First Name', surnameLabel: 'Last Name', emailLabel: 'Email Address', phoneLabel: 'Phone Number', companyLabel: 'Company Name', jobTitleLabel: 'Job Title', countryLabel: 'Country', countryPlaceholder: 'Select country...', zipLabel: 'ZIP/Postal Code', commentsLabel: 'Comment', submitLabel: 'Submit',
```

Comment:
Add line breaks after each `,`

Reply:
@solocommand This should have been caught by the linter and failed the build... any thoughts why it didn't?